LICENSE.TXT

```diff
@@ -0,0 +1,21 @@
+Copyright 2020 (c) 2020 Raspberry Pi (Trading) Ltd.
```

Comment:
2021 ?

Reply:
seems to be what all the others have; go figure

Reply:
Oh, so they do! I never noticed that, oops.